staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go

```diff
@@ -1067,44 +1056,6 @@ func TestDifferentFlowsWithoutQueuing(t *testing.T) {
 	}.exercise(t)
 }
 
-func TestTimeout(t *testing.T) {
```

Comment:
Do we have other tests here that exercise the newly added rejections?

Reply:
yes, the existing `TestContextCancel` test exercises the request context timeout scenario:
https://github.com/kubernetes/kubernetes/blob/3ed537ab46af2db3a5897439fd40b40723de5cbe/staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go#L1123

Reply:
@wojtek-t : in other words, there are no added rejections in the PR, it only removes.